mis_builder/models/mis_builder.py

```diff
@@ -0,0 +1,913 @@
+            return False
+
+    def _fetch_queries(self, cr, uid, c, context):
+        res = {}
+        report = c.report_instance_id.report_id
+        for query in report.query_ids:
+            obj = self.pool[query.model_id.model]
+            eval_context = {
+                'time': time,
+                'datetime': datetime,
+                'dateutil': dateutil,
+                # deprecated
+                'uid': uid,
+                'context': context,
+            }
+
+            if not c.date_from or not c.date_to:
+                raise orm.except_orm(_('Error!'),
+                                     _('Please define From and To dates for '
+                                       'period %s.') % c.name)
+            domain = query.domain and \
+                safe_eval(query.domain, eval_context) or []
+            domain.extend(self._get_additional_query_filter(
+                cr, uid, c.id, query, context=context))
+            if query.date_field.ttype == 'date':
+                domain.extend([(query.date_field.name, '>=', c.date_from),
+                               (query.date_field.name, '<=', c.date_to)])
+            else:
+                tz = context.get('tz', False) or 'UTC'
+                datetime_from = _utc_midnight(
+                    c.date_from, tz)
+                datetime_to = _utc_midnight(
+                    c.date_to, tz, add_day=1)
+                domain.extend([(query.date_field.name, '>=', datetime_from),
+                               (query.date_field.name, '<', datetime_to)])
+            if obj._columns.get('company_id', False):
+                domain.extend(['|', ('company_id', '=', False),
+                               ('company_id', '=', c.company_id.id)])
+            field_names = [f.name for f in query.field_ids]
+            if not query.aggregate:
+                obj_ids = obj.search(cr, uid, domain, context=context)
+                data = obj.read(
+                    cr, uid, obj_ids, field_names, context=context)
+                res[query.name] = [AutoStruct(**d) for d in data]
+            elif query.aggregate == 'sum':
+                data = obj.read_group(
+                    cr, uid, domain, field_names, '', context=context)
+                s = AutoStruct(count=data[0]['_count'])
+                for field_name in field_names:
+                    v = data[0][field_name]
+                    setattr(s, field_name, v)
+                res[query.name] = s
+            else:
+                obj_ids = obj.search(cr, uid, domain, context=context)
+                data = obj.read(
+                    cr, uid, obj_ids, field_names, context=context)
+                s = AutoStruct(count=len(data))
+                if query.aggregate == 'min':
+                    agg = min
+                elif query.aggregate == 'max':
+                    agg = max
+                elif query.aggregate == 'avg':
+                    agg = lambda l: sum(l) / float(len(l))
```

Comment:
@sebalix this was a backport issue. It should be fixed with @jbeficent's last commit: https://github.com/OCA/account-financial-reporting/commit/30744ab287a634a189d2f51b7ee88bed93475686